crates/client/src/api.rs

```diff
@@ -219,57 +219,43 @@ impl Client {
     /// Gets the latest checkpoint from the registry.
     pub async fn latest_checkpoint(
         &self,
+        registry_domain: Option<&RegistryDomain>,
     ) -> Result<SerdeEnvelope<TimestampedCheckpoint>, ClientError> {
         let url = self.url.join(paths::fetch_checkpoint());
-        tracing::debug!("getting latest checkpoint at `{url}`");
+        tracing::debug!(
+            desc = "getting latest checkpoint",
+            url = url,
+            registry_header = ?registry_domain
+        );
```

Comment:
There is a pretty strong convention for `message` here; actually more than a convention as some tracing-subscribers treat it specially.

```suggestion
        tracing::debug!(
            message = "getting latest checkpoint",
            url,
            registry_header = ?registry_domain
        );
```
or exactly equivalent (a trailing string is implicitly assigned to `message`):
```suggestion
        tracing::debug!(
            url,
            registry_header = ?registry_domain
            "getting latest checkpoint",
        );
```

Reply:
Cool, I'll revise. Thanks for explaining the convention.

Reply:
Fixed